manifests/k8s-spot-termination-handler/k8s-spot-termination-handler.yaml

```diff
@@ -66,7 +66,7 @@ spec:
       serviceAccountName: k8s-spot-termination-handler
       containers:
         - name: k8s-spot-termination-handler
-          image: stafot/kube-spot-termination-notice-handler:1.21.0_628d3aef
+          image: stafot/kube-spot-termination-notice-handler:1.21.0_822b6001
```

Comment:
@stafot at some point we need to move this in the mattermost registry

Reply:
I will. transform it to helm really soon